backend/server.js

```diff
@@ -17,7 +17,9 @@ let expectedAudience = `/projects/${process.env.PROJECT_NUMBER}/apps/${process.e
 
 let app = express();
 
-app.use(checkIap);
+if(process.env.USE_IAP === "true") {
```

Comment:
It'd be safer to flip this and check for != false. If something gets messed up with env variables, we always want to default to using IAP

Reply:
great shout. Will do.